contracts-periphery/src/UmbraBatchSend.sol

```diff
@@ -24,69 +27,133 @@ contract UmbraBatchSend {
     bytes32 ciphertext;
   }
 
+  mapping(address => uint256) internal TotalTransferAmountPerToken;
```

Comment:
nit: should start with a lowercase 't' `totalTransferAmountPerToken`

Reply:
gotcha https://github.com/ScopeLift/umbra-protocol/blob/b156045ea6b0d75c49dbb6500e5e9e6835131723/contracts-periphery/src/UmbraBatchSend.sol#L30